### DISCUSSION TITLE
Download Azure Data Studio - Windows System Installer not working

### DISCUSSION THREAD

**Discussion by arwisdom** [microsoft/azuredatastudio#15813] [General]
Download Azure Data Studio - Windows System Installer not working 
Seems to keep download Linux rpm file.

**Comment by Charles-Gagnon** [comment]
Could you provide the link you're using for this? The ones on https://github.com/microsoft/azuredatastudio and https://docs.microsoft.com/en-us/sql/azure-data-studio/download-azure-data-studio?view=sql-server-ver15 seem to work as expected for me.

**Comment by arwisdom** [comment]
This is working today..

Was not working yesterday, I also asked my colleagues to try..

I have not updated our BI TS servers.

Many Thanks
Andrew

From: Charles Gagnon ***@***.***>
Sent: 19 June 2021 07:00
To: microsoft/azuredatastudio ***@***.***>
Cc: Wisdom, Andrew ***@***.***>; Author ***@***.***>
Subject: :EXTERNAL: Re: [microsoft/azuredatastudio] Download Azure Data Studio - Windows System Installer not working (#15813)


Could you provide the link you're using for this? The ones on https://github.com/microsoft/azuredatastudio<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmicrosoft%2Fazuredatastudio&data=04%7C01%7Cawisdom%40harsco.com%7Ca40a19343c604e151c0808d932e78904%7C320c95f36f404b25aa055c3137575fe9%7C0%7C0%7C637596792297173295%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=5YjEM4ASiG6m7Udws%2BqNwQpiCEe97EIqV%2BlsZCL8G5o%3D&reserved=0> and https://docs.microsoft.com/en-us/sql/azure-data-studio/download-azure-data-studio?view=sql-server-ver15<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdocs.microsoft.com%2Fen-us%2Fsql%2Fazure-data-studio%2Fdownload-azure-data-studio%3Fview%3Dsql-server-ver15&data=04%7C01%7Cawisdom%40harsco.com%7Ca40a19343c604e151c0808d932e78904%7C320c95f36f404b25aa055c3137575fe9%7C0%7C0%7C637596792297183245%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=EJU7x6YWbjZOR6KXJh%2FPOhogQrEE%2Fp8kBxTw8pYBPos%3D&reserved=0> seem to work as expected for me.

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmicrosoft%2Fazuredatastudio%2Fdiscussions%2F15813%23discussioncomment-890708&data=04%7C01%7Cawisdom%40harsco.com%7Ca40a19343c604e151c0808d932e78904%7C320c95f36f404b25aa055c3137575fe9%7C0%7C0%7C637596792297183245%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=GqE5stVZle5vVCg44mQl3PwBlKDCz0auaNySUzgv9jA%3D&reserved=0>, or unsubscribe<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAIGGVEP2TKO7I6SQMF2SZEDTTQW7VANCNFSM465M4EBQ&data=04%7C01%7Cawisdom%40harsco.com%7Ca40a19343c604e151c0808d932e78904%7C320c95f36f404b25aa055c3137575fe9%7C0%7C0%7C637596792297183245%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=9gLWLQJLBTJAjHV1vCF70qZAhJUhG%2FPG7%2FIrDJuVzdM%3D&reserved=0>.

 CAUTION : This email originated from outside of the organization. Do not click links or open attachments unless you recognize the sender and know the content is safe.

**Comment by dzsquared** [comment]
Hi folks - confirming that we did have an error in the docs yesterday morning, but this was corrected around 10:30am PDT. https://github.com/MicrosoftDocs/sql-docs/issues/6518
